app/src/views/OperationalLearning/LearningMap/utils.ts

```diff
@@ -0,0 +1,82 @@
+import type {
+    CircleLayer,
+    CirclePaint,
+    FillLayer,
+} from 'mapbox-gl';
+
+import {
+    COLOR_BLACK,
+    COLOR_BLUE,
+    COLOR_DARK_GREY,
+    COLOR_LIGHT_GREY,
+    COLOR_RED,
+    COLOR_YELLOW,
+} from '#utils/constants';
+
+const COLOR_LEARNING_AND_PERSONNEL = COLOR_BLUE;
+const COLOR_LEARNING_ONLY = COLOR_RED;
+const COLOR_PERSONNEL_ONLY = COLOR_YELLOW;
+const COLOR_DEFAULT = COLOR_BLACK;
+
+export const adminFillLayerOptions: Omit<FillLayer, 'id'> = {
+    type: 'fill',
+    paint: {
+        'fill-color': [
+            'case',
+            ['boolean', ['feature-state', 'hovered'], false],
+            COLOR_DARK_GREY,
+            COLOR_LIGHT_GREY,
+        ],
+    },
+};
+
+const circleColor: CirclePaint['circle-color'] = [
+    'case',
+    COLOR_LEARNING_AND_PERSONNEL,
+    COLOR_LEARNING_ONLY,
+    COLOR_PERSONNEL_ONLY,
+    COLOR_DEFAULT,
+];
+
+const basePointPaint: CirclePaint = {
+    'circle-radius': 5,
+    'circle-color': circleColor,
+    'circle-opacity': 0.8,
+};
+
+export const basePointLayerOptions: Omit<CircleLayer, 'id'> = {
+    type: 'circle',
+    paint: basePointPaint,
+};
+
+const baseOuterCirclePaint: CirclePaint = {
+    'circle-color': circleColor,
+    'circle-opacity': 0.4,
+};
+
+const outerCirclePaintForPersonnel: CirclePaint = {
+    ...baseOuterCirclePaint,
+    'circle-radius': [
+        'interpolate',
+        ['linear', 1],
+        ['get', 'learning'],
+
+        2,
+        5,
+        4,
+        7,
+        6,
+        9,
+        8,
+        11,
+        10,
+        13,
+        12,
+        15,
+    ],
+};
```

Comment:
if we are thinking of making the circles bigger based on learning count then we need to appropriately use the logic for defining stops.